### PR TITLE
Relative path refactoring

### PR DESCRIPTION
**Pull request by nathanmartins**

### CODE REVIEW

**Review by rhuankarlus — CHANGES_REQUESTED** [review]

**Review by rhuankarlus — CHANGES_REQUESTED** [review]

File: aurum/base.py [hunk 1]
```diff
@@ -298,8 +307,7 @@ def end_experiment() -> bool:
         requirements_metadata = RequirementsMetaData().get_latest() or RequirementsMetaData()
         dataset_metadata = DatasetMetaData().get_latest() or DatasetMetaData()
         code_metadata = CodeMetaData().get_latest() or CodeMetaData()
-        destination = os.path.join(git.get_git_repo_root(), cons.REPOSITORY_DIR, cons.EXPERIMENTS_METADATA_DIR,
-                                   f"{theorem.experiment_id}.json")
+        destination = os.path.join(cons.REPOSITORY_DIR, cons.EXPERIMENTS_METADATA_DIR, f"{theorem.experiment_id}.json")
```
[the remark this answers is not in the document]
Comment (nathanmartins) [reply aurum/base.py:311]: Resolved.

File: aurum/commands.py [hunk 2]
```diff
@@ -130,11 +143,7 @@ def run_rm(parsed_result: argparse.Namespace) -> None:
 def run_load(parsed_result: argparse.Namespace, skip_package_install: bool = False) -> None:
     logging.info(f"Attempting to load experiment with tag: {parsed_result.tag}")
 
-    experiment_dir = os.path.join(
-        git.get_git_repo_root(),
-        cons.REPOSITORY_DIR,
-        cons.EXPERIMENTS_METADATA_DIR,
-    )
+    experiment_dir = os.path.join(cons.REPOSITORY_DIR, cons.EXPERIMENTS_METADATA_DIR)
```
[the remark this answers is not in the document]
Comment (nathanmartins) [reply aurum/commands.py:146]: Resolved.

File: aurum/commands.py [hunk 3]
```diff
@@ -144,8 +153,7 @@ def run_load(parsed_result: argparse.Namespace, skip_package_install: bool = Fal
 
     emd = ExperimentMetaData(os.path.join(experiment_dir, f"{parsed_result.tag}.json"))
 
-    requirements_metadata_dir = \
-        os.path.join(git.get_git_repo_root(), cons.REPOSITORY_DIR, cons.REQUIREMENTS_METADATA_DIR)
+    requirements_metadata_dir = os.path.join(cons.REPOSITORY_DIR, cons.REQUIREMENTS_METADATA_DIR)
```
[the remark this answers is not in the document]
Comment (nathanmartins) [reply aurum/commands.py:156]: Resolved.

File: aurum/dataset_tracker.py [hunk 4]
```diff
@@ -12,46 +12,50 @@ def __init__(self) -> None:
...
+            dsm, _ = DatasetMetaData().get_by_ds_name(ds, cwd=cwd)
             full_hash_str += dsm.file_hash
 
         full_hash = hashlib.sha1()
         full_hash.update(full_hash_str.encode())
         return full_hash.hexdigest()
 
 
-def use_datasets(*args):
+def use_datasets(*args, cwd: str, ):
```
[the remark this answers is not in the document]
Comment (nathanmartins) [reply aurum/dataset_tracker.py:27]: Good idea, did that, resolved.

**Pull request merged**